macros/tests.sql

```diff
@@ -146,10 +146,9 @@
       {{ dbt_unit_testing.debug(test_query) }}
     {% endif %}
 
-    {% set r1 = run_query("select count(1) from (" ~ expectations_query ~ ") as t") %}
+    {% set r1 = run_query("select * FROM (select count(1) as expectation_count from (" ~ expectations_query ~ ")), (select count(1) as actual_count from (" ~ actual_query ~ "))") %}
```

Comment:
This query doesn't run in Postgres/snowflake. It requires a couple of aliases that we'll not use...
```
select * FROM (select count(1) as expectation_count from (" ~ expectations_query ~ ") as exp) as exp_count, (select count(1) as actual_count from (" ~ actual_query ~ ") as act) as act_count 
```

Reply:
Should be fixed now!